content/en/cloudcraft/getting-started/live-vs-snapshot-diagrams.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Live vs Snapshot Diagrams
+---
+
+## Overview
+
+Cloudcraft offers two types of diagrams for visualizing your cloud infrastructure: Live and Snapshot. This document explains the key differences between these diagram types and how to use them effectively.
+
+## Live diagrams
+
+Live diagrams provide real-time representation of your AWS or Azure infrastructure. Key features include:
+
+- **Automatic updates**: The diagram reflects changes in your infrastructure when you access it.
+- **Real-time accuracy**: Ensures your diagram always represents the current state of your cloud infrastructure.
+- **Read-only**: Prevents accidental modifications to your infrastructure layout.
+
+## Snapshot diagrams
+
+Snapshot diagrams offer a static view of your infrastructure that you can modify to suit your visual needs. Important characteristics include:
+
+- **Editable layout**: Allows you to move components and adjust the diagram's design.
+- **Custom additions**: You can add components from the Design tab.
+- **Static representation**: The diagram doesn't update automatically to reflect infrastructure changes.
+
+## Key differences
+
+### Updating mechanism
+
+- **Live diagrams**: Update automatically when accessed, reflecting any changes in your AWS or Azure infrastructure.
+- **Snapshot diagrams**: Remain static and don't update automatically.
+
+### Editing capabilities
+
+- **Live diagrams**: Read-only prevents modifications to the diagram.
```

Comment:
Maybe we can say you can only modify your query / group bys instead of saying "read-only"?